### ISSUE TITLE
Enrollment not showing up in attendance sheet, but appears in the class offering.

### ISSUE THREAD

**Issue by thekarinoelle** [endgamesimprov/EgSites#64]
I found an additional one to the one Dan reported. Here is all of the screen captures.

**Skinner compared to another classmate on individual registration page:**
![Screen Shot 2019-12-05 at 3 45 06 PM](https://user-images.githubusercontent.com/56705710/70284120-cca2f480-1777-11ea-88ee-75e0f64ffdc3.png)
![Screen Shot 2019-12-05 at 3 45 24 PM](https://user-images.githubusercontent.com/56705710/70284121-cca2f480-1777-11ea-9c98-e2df97629bb0.png)
![Screen Shot 2019-12-05 at 3 45 33 PM](https://user-images.githubusercontent.com/56705710/70284122-cca2f480-1777-11ea-9f75-9919febbb5ac.png)

**Attendance sheets for both Nov 21st and Dec 4th without Skinner**
![Screen Shot 2019-12-05 at 3 46 23 PM](https://user-images.githubusercontent.com/56705710/70284151-ee9c7700-1777-11ea-951e-5a608d2ffada.png)
![Screen Shot 2019-12-05 at 3 46 29 PM](https://user-images.githubusercontent.com/56705710/70284152-ee9c7700-1777-11ea-9186-884656df4247.png)

**Stusiak, Tim shows up on offering page registrations, but not on the attendance page**
![Screen Shot 2019-12-05 at 3 57 56 PM](https://user-images.githubusercontent.com/56705710/70284295-6ff40980-1778-11ea-85d0-44deebd5b73f.png)
![Screen Shot 2019-12-05 at 3 58 08 PM](https://user-images.githubusercontent.com/56705710/70284305-72eefa00-1778-11ea-90b1-18526497779c.png)
![Screen Shot 2019-12-05 at 3 46 29 PM](https://user-images.githubusercontent.com/56705710/70284322-75515400-1778-11ea-80f7-5892c9d2ebad.png)

**Comment by thekarinoelle** [comment]
Purchase dates:
Nov 12th and Nov 13th.

**Comment by endgamesimprov** [comment]
Also, registra-690c-4619-882a-7ff3fbc07677

**Comment by endgamesimprov** [comment]
Well, i have to warn you. This isn't looking good. I'm finding hundreds. Stay tuned.

**Comment by endgamesimprov** [comment]
The good news is that it stopped happening around Nov 13th. Which is a real good sign. But I'm wondering why more people haven't caught these. But still investigating!

**Comment by endgamesimprov** [comment]
Ok scratch that. I might have just found another bug. :(

**Comment by thekarinoelle** [comment]
aw man. not sure if this is helpful, but I discovered that it's still possible to update Tim S's attendance by clicking on his individual attendance sheet.

**Comment by scottendgames** [comment]
Just putting these here for my own sake:
registra-690c-4619-882a-7ff3fbc07677 2019-11-13 02:36:51
registra-09e1-465b-9bab-104cae8e7191 2019-11-13 10:33:50
registra-865b-491c-98ee-29ee3d07efea 2019-11-12 09:22:12

So the good news is the issue we found is isolated to these three. The 100's i found before seem to be a separate bug which has to do with a bunch of "OffDays" not being created for people. But I think that was from the conversion, because it stopped the weekend after conversion. I'll look into it and see if I want to correct it. 

The other good news is that the three that appear are all from the weekend of the conversion. So either I fixed something and it solved it, or there was some issue that occurred because of the conversion that I didn't catch. 

My next step is to correct them by hand. But now I have a system to monitor this issue. If it occurs again, we can hopefully catch it and understand why it occurred.
